### PR TITLE
Gherkin stories for the Windows setup logging

### PR DESCRIPTION
**Pull request by mrpau-richard**
### Summary
I created a Gherkin stories for this [PR](https://github.com/learningequality/kolibri/pull/5732)
This is still a work in progress PR

### Reviewer guidance

…

### References

…

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: integration_testing/features/super-admin/super-admin-windows-installer-setup-error-log.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Feature: Super admin Windows installer setup failed show the error logs
```
Comment (cpauya) [inline integration_testing/features/super-admin/super-admin-windows-installer-setup-error-log.feature]: Please re-phrase, maybe into: `Super admin can see error logs during a failed Windows installer setup.`

File: integration_testing/features/super-admin/super-admin-windows-installer-setup-error-log.feature [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    Super admin needs to be able to see the error logs when the Kolibri Windows installation failed
+
+    Background:
+        Given that I have the Kolibri Windows installer
+          And I am using a Windows environment
+
+    Scenario: Install the kolibri Windows installer with the error
+        And I double click the Kolibri Windows installer
```
Comment (cpauya) [inline integration_testing/features/super-admin/super-admin-windows-installer-setup-error-log.feature]: I think this is missing a `Given` right after the `Scenario` line?

File: integration_testing/features/super-admin/super-admin-windows-installer-setup-error-log.feature [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        And I see the setup message box that Python is required to install
+        And I click "yes" to install Python
+       Then I see the Python is installing
+        And I manually delete or rename the "pip.exe" in the "C:\Python34\Script" path
+        And I continue the Kolibri installation
+        And the kolibri error message show up
+       Then I see the kolibri error dialog box
+        When I click the Kolibri log link
+       Then I see all the installation error logs
+
```
Comment (cpauya) [inline integration_testing/features/super-admin/super-admin-windows-installer-setup-error-log.feature:20]: Can we also put other scenarios which may possibly fail during a Windows installation?  Maybe like:

1. Install using `KOLIBRI_HOME_PORT` environment variable but with a non-numeric value like `ABC` or a combo like `ABC8080`?
1. Upgrade from an existing Kolibri installation where we want to preserve the database?

**Review by cpauya — CHANGES_REQUESTED** [review]

File: integration_testing/features/super-admin/super-admin-windows-installer-setup-error-log.feature (hunk 3 above)
Comment (mrpau-richard) [reply integration_testing/features/super-admin/super-admin-windows-installer-setup-error-log.feature:20]: > Install using KOLIBRI_HOME_PORT environment variable but with a non-numeric value like ABC or a combo like ABC8080?

I think we can't add that scenarios here, The error happens after the Kolibri setup successfully install not during the Kolibri setup installation. I created an issue about this [here](https://github.com/learningequality/kolibri-installer-windows/issues/113), lets create a gherkin feature after that issue resolve


> Upgrade from an existing Kolibri installation where we want to preserve the database?

That scenario also not related to this gherkin feature. During the installation the Kolibri setup did not check if there is an existing database.

File: integration_testing/features/super-admin/super-admin-windows-installer-setup-error-log.feature [hunk 4]
```diff
@@ -0,0 +1,23 @@
+Feature: Super admin can see error logs during a failed Windows installer setup
+    Super admin needs to be able to see the error logs when the Kolibri Windows installation failed
+
+    Background:
+        Given that I have the Kolibri Windows installer
+          And I am using a Windows environment
+          And I double click the Kolibri Windows installer
+
+    Scenario: Install the kolibri Windows installer with the error
```
Comment (cpauya) [inline integration_testing/features/super-admin/super-admin-windows-installer-setup-error-log.feature:9]: This `Scenario` still has no `Given` or `When` statements after it.  Is this intentional?

File: integration_testing/features/super-admin/super-admin-windows-installer-setup-error-log.feature [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+        And I see the setup message box that Python is required to install
+        And I click "yes" to install Python
+       Then I see the Python is installing
+        And I manually delete or rename the "pip.exe" in the "C:\Python34\Script" path
+        And I continue the Kolibri installation
+        And the kolibri error message show up
+       Then I see the kolibri error dialog box
+        When I click the Kolibri log link
+       Then I see all the installation error logs
+
```
Comment (cpauya) [inline integration_testing/features/super-admin/super-admin-windows-installer-setup-error-log.feature:20]: Please fix the indentation and capitalisations in this `Scenario`.  I remember that we're using two spaces for to indent Gherkin statements.

**Review by cpauya — CHANGES_REQUESTED** [review]
Good update, very straight-forward.  Please address my requested changes.

**Review by cpauya — APPROVED** [review]
Good updates!  Looks good to me. 😸

**Comment by indirectlylit** [comment]
fixes https://github.com/learningequality/kolibri/issues/5911

**Pull request merged**